deploy/cr.yaml

```diff
@@ -105,7 +105,7 @@ spec:
 #          test-label: value
 
 
-  image: perconalab/percona-postgresql-operator:main-ppg16-postgres
+  image: percona/percona-postgresql-operator:2.4.0-ppg16-postgres
```

Comment:
hi, @ptankov, please add a minor version of PG to tag as well. As I remember you even have a task for it.

Reply:
done